frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -34,16 +32,39 @@ public Car getCar(@PathVariable("carId") int carId) {
     @GetMapping(path = "/cars/{list}")
     public List<Car> getBatchCars(@PathVariable List<String> list) {
         List<Car> newList = new ArrayList<>();
-        for(String entry : list) {
-         newList.add(map.get(entry));
+        for (String entry : list) {
+            newList.add(map.get(entry));
         }
         return newList;
     }
+
     @PostMapping(path = "/cars", consumes = {"application/json"})
     public Map<String, Car> createCarList(@RequestBody List<Car> carList) {
         carNewList = carList;
         for (int i = 0; i < carList.size(); i++) {
             map.put(String.valueOf(i), carNewList.get(i));
+            carNewList.get(i).setId(i);
+        }
```

Comment:
Would the Car Id be better as an Integer object rather than a String? Looks like it will alway be a number

Reply:
resolved